core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -83,6 +89,11 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
   def main(args: Array[String]) {
 
     info("Starting mirror maker")
+    println("WARNING: The default partition assignment strategy of mirror maker will change from 'range' to 'roundrobin' " +
```

Comment:
@hachikuji Thanks for the explanation. I agree that stopping and restarting everything for an upgrade is not ideal. I'm thinking about scenarios with multiple MM processes (several `kafka.tools.MirrorMaker` instances) and I can think of when we would want to mirror a subset of topics in each MM process, with the most granular being one topic per MM process (which wouldn't really be affected by this change). So I'm guessing one or both of the following is possible (even though I'm not sure how):
- a MM process mirroring some partitions of a topic (and not all of its partitions)
- multiple MM processes mirror the same message log (in parallel)

Is that so? Or I'm missing something? Thanks.